metricbeat/module/galera/_meta/config.yml

```diff
@@ -0,0 +1,30 @@
+- module: galera
+  metricsets: ["status"]
+  enabled: false
+  period: 10s
+
+  # Specifies the status variables to query from the cluster
```

Comment:
The short config should not contain all the config options. Please also create a `config.reference.yml` file.